circuits/cpp/barretenberg/.github/workflows/publish.yml

```diff
@@ -71,7 +110,8 @@ jobs:
           yarn install && yarn && yarn build
 
       - name: Tar and GZip barretenberg.wasm
-        run: tar -cvzf barretenberg.wasm.tar.gz cpp/build-wasm/bin/barretenberg.wasm
+        working-directory: cpp/build-wasm/bin
+        run: tar -cvzf barretenberg.wasm.tar.gz barretenberg.wasm
 
       - name: Setup Node.js
```

Comment:
Do we need that ? This should be already available on image

Reply:
on image?

Reply:
We currently use this for blackbox function solving, we natively download this from the releases and it will polyfill the pedersen, schnorr and scalar mul functionality

Reply:
Though its always been there (previously because for windows builds we were using it for proving and verifying) -- probably okay to leave it as part of the build artifacts

Reply:
Oh you mean node, when I ran it, it kept saying it couldn't find it